push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/APNsPushNotificationSender.java

```diff
@@ -209,4 +209,43 @@ public void messageSendFailed(ApnsNotification message, Throwable e) {
         // null if, why ever, there was no cert/passphrase
         return null;
     }
+
+    /**
+     * Configure the Gateway to the Apns servers.
+     * Default gateway and port can be override with respectively :
+     *  - aerogear.apns.push.host
+     *  - aerogear.apns.push.port
+     *
+     * Feedback gateway and port can be override with  respectively :
+     *  - aerogear.apns.feedback.host
+     *  - aerogear.apns.feedback.port
+     * @param iOSVariant
+     * @param builder
+     */
+    private void configureDestinations(iOSVariant iOSVariant, ApnsServiceBuilder builder) {
+        // pick the destination:
+        if (iOSVariant.isProduction()) {
+            builder.withProductionDestination();
+        } else {
+            builder.withSandboxDestination();
+        }
+
+        //Is the gateway host&port provided by a system property ?
+        if(System.getProperty(AEROGEAR_APNS_PUSH_HOST)!= null){
```

Comment:
This will throw a `SecurityException` if a security manager exists and its `checkPropertyAccess()` method doesn't allow access to the specified system property.

In the past I have done these calls inside of try/catch blocks, like:
```
try {
    someProperty = System.getProperty(KEY);
    } catch (Throwable t) {
...
}
```